### PR TITLE
rework live patching

### PR DESCRIPTION
**Pull request by mcalmer**
Split out spacewalk-manage-channel-lifecycle into an own Setup Chapter and change the order a bit.

The idea:

1st prepare the channels for Live Patching either with CLM (prefered) or with spacewalk-manage-channel-lifecycle tool.
2nd assign the clients to the clones and prepare the system for Live Patching.

### CODE REVIEW

**Comment by mcalmer** [comment]
> (Assuming no content changes)

The majority of the text hasn't changed, but on top of the sections there is a bit of new or changed content.

File: modules/administration/pages/live-patching-channel-setup.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+1. Use Content Lifecycle Management to clone the product tree and remove kernel versions newer than the running one. This procedure is explained in the xref:content-lifecycle-examples.adoc#enhance-project-with-livepatching[Content Livecycle Management Examples].
+This is the prefered solution.
+
+2. Use `spacewalk-manage-channel-lifecycle` tool. This procedure is more manual and requires commandline tools as well as the Web UI. This procedure is explained below for SLES 15 SP1, but it works similar for SLE12 SP4 or later as well.
+
+== Use spacewalk-manage-channel-lifecycle for Live Patching
+
+Cloned vendor channels should be prefixed by ``dev`` for development, ``testing``, or  ``prod`` for production.
+In this procedure, you will create a ``dev`` cloned channel, and later, you will need to promote the channel to ``testing``.
```
Comment (Loquacity) [inline modules/administration/pages/live-patching-channel-setup.adoc]:
```suggestion
In this procedure, you will create a ``dev`` cloned channel and then promote the channel to ``testing``.
```

File: modules/administration/pages/live-patching-channel-setup.adoc [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+.Procedure: Cloning Live Patching Channels
+
+. At the command prompt on the client, as `root`, obtain the current package channel tree:
```
Comment (Loquacity) [inline modules/administration/pages/live-patching-channel-setup.adoc]:
```suggestion
. At the command prompt on the client, as root, obtain the current package channel tree:
```

File: modules/administration/pages/live-patching-sles15.adoc [hunk 3]
```diff
@@ -10,6 +10,7 @@ Before you begin, ensure:
 * Your SLES{nbsp}15 Salt clients are registered with {productname}
 * You have access to the SLES{nbsp}15 channels appropriate for your architecture, including the Live Patching child channel (or channels)
 * The clients are fully synchronized
+* Assign the clients to the cloned channels prepared for Live Patching (explained before)
```
Comment (Loquacity) [inline modules/administration/pages/live-patching-sles15.adoc]:
```suggestion
* Assign the clients to the cloned channels prepared for live patching
```

**Comment by Loquacity** [comment]
> > (Assuming no content changes)
> 
> The majority of the text hasn't changed, but on top of the sections there is a bit of new or changed content.

OK, I just did an editing pass. Sorry for not noticing that earlier.

File: modules/administration/pages/live-patching-channel-setup.adoc [hunk 4]
```diff
@@ -0,0 +1,76 @@
+[[live-patching-channel-setup]]
+= Setup Channels for Live Patching
+
+Every update of the full kernel package would require a reboot.
+Therefor it is important to not have newer kernels available in the channels assigned to clients which should use Live Patching.
+
+For these clients only the Live Patching channel contains updates of the running kernel.
```
[the remark this answers is not in the document]
Comment (mcalmer) [reply modules/administration/pages/live-patching-channel-setup.adoc]: This does not sound right either. I will chang it to:

```
Clients using live patching have updates for the running kernel in the live patching channels.
```

Please verify.

**Pull request merged**